src/stability_sdk/animation.py

```diff
@@ -0,0 +1,1122 @@
+import base64
+import bisect
+import cv2
+import glob
+import json
+import logging
+import math
+import numpy as np
+import os
+import param
+import random
+import shutil
+import subprocess
+
+from collections import OrderedDict, deque
+from dataclasses import dataclass, fields
+from keyframed.dsl import curve_from_cn_string
+from PIL import Image, ImageFilter, ImageOps
+from types import SimpleNamespace
+from typing import cast, Deque, Generator, List, Optional, Tuple, Union
+
+from stability_sdk.api import Context, generation
+from stability_sdk.utils import (
+    camera_pose_transform,
+    color_adjust_transform,
+    depth_calc_transform,
+    guidance_from_string,
+    image_mix,
+    image_to_png_bytes,
+    interpolate_mode_from_string,
+    resample_transform,
+    sampler_from_string,
+)
+import stability_sdk.matrix as matrix
+
+logger = logging.getLogger(__name__)
+logger.setLevel(level=logging.INFO)
+
+DEFAULT_MODEL = 'stable-diffusion-v1-5'
```

Comment:
upgrade to `stable-diffusion-xl-beta-v2-2-2`

Reply:
This is intentional for animation to default to v1-5 but generation to default to SDXL. People can switch the model over to SDXL but it costs more than the documented baseline defaults.